pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -486,7 +492,7 @@ func getMappedState(state kb.State) State {
 	case kb.WARN:
 		return Fail
 	case kb.INFO:
-		return Fail
+		return NotApplicable
```

Comment:
Why do we need this change?
Probably we need something like:
```
case NOTAPPLICABLE:
    return NotApplicable
```

Reply:
kb.INFO state is set by kube-bench in the scan output for any tests that are marked as type: "skip"
Thus now that all NA and default skip from KDM are marked as type: "skip", the kube-bench output will mark these tests state as kb.INFO. refer https://github.com/aquasecurity/kube-bench#omitting-checks

Summarizer will need to translate this to NotApplicable so that it shows as NA in the CIS v2 report.